pegasus/sites.v3/code.org/views/teach_page_labs.haml

```diff
@@ -0,0 +1,53 @@
+%h2
+  =hoc_s(:"teach_labs_section.header")
+%p
+  =hoc_s(:"teach_labs_section.description")
+.action-block__wrapper.action-block__wrapper--three-col
+  .action-block.action-block--one-col.flex-space-between
+    .content-wrapper 
+      %p.overline=hoc_s(:"teach_labs_section.spritelab.overline")
+      %h3=hoc_s(:"teach_labs_section.spritelab.title")
+      %img{src: "/images/build-your-way-spritelab.png", alt: ""}
+      %p
+        %strong=hoc_s(:"teach_labs_section.label_what_you_can_make")
+        =hoc_s(:"teach_labs_section.spritelab.what_you_can_make")
+      %p
+        %strong=hoc_s(:"teach_labs_section.label_experience")
+        =hoc_s(:"teach_labs_section.spritelab.experience")
+    .content-footer 
+      %a.link-button{href: "/educate/spritelab", aria:{label: hoc_s(:"teach_labs_section.spritelab.aria_label_learn_more")}}
+        =hoc_s(:"teach_labs_section.spritelab.call_to_action_learn_more")
+      %a.link-button.secondary{href: CDO.studio_url("/projects/spritelab/new")}
```

Comment:
Oops just noticed this after I submitted my review. Should these buttons be flipped? "Try …" is above "Learn more". Or is it like this because the audience of this page is teachers?

Reply:
I'm not sure I understand your question? The "Learn More" button is on top both in this code and in the [figma](https://www.figma.com/design/9HOagNHsLMZUVOJnB9CqOz/Progress-Launch?node-id=2305-1214&m=dev)

Reply:
Ah ok, I couldn't find the Figma. Usually we put the more active call to action on top, but this makes sense considering teachers probs care more about learning more than actually doing the activity.

Reply:
Sorry, should've linked the Figma in the description 😅